### ISSUE TITLE
Add ability to set crontab env vars via server.crontab (or a separate operation)

### ISSUE THREAD

**Issue by taranlu-houzz** [Fizzadar/pyinfra#526]
**Is your feature request related to a problem? Please describe.**
Currently, `server.crontab` is unable to set env vars within the crontab itself. Cron env entries take this form: `ENV_VAR=VALUE` and are very useful for doing things like setting the `PATH` (since cron executes in a sterile environment). I believe order also matters, so typically the env vars will be declared at the top of the crontab file.

**Describe the solution you'd like**
Either a parameter that takes a dict of key value pairs to use in the cron tab, or a separate operation to handle that.

**Comment by Fizzadar** [comment]
This can currently be done by adding them to the `command` itself, ie `server.crontab(command='PATH=/somewhere echo mycommand')`. There's room here for an additional `command_env`/similar argument that would modify the command itself, if desired?

There's also "global" environment variables supported by some cron implementations ([see this SO question](https://stackoverflow.com/questions/2229825/where-can-i-set-environment-variables-that-crontab-will-use)) but since this is specific to certain distributions I don't think it's suitable here, maybe as a separate `server.crontab_env` operation or similar.

**Comment by taranlu-houzz** [comment]
Yeah, that is what I ended up doing. I didn't realize that global env vars are distribution dependent. I do still think that an additional operation would be helpful, since it is a useful feature to have access to.

**Comment by Fizzadar** [comment]
Closing as stale, crontab has been rewritten in the last ~year!
